iOS/MAX Unreal Plugin/MAUnrealPlugin.mm

```diff
@@ -377,21 +384,28 @@ - (void)setUserId:(NSString *)userId
 
 - (void)setMuted:(BOOL)muted
 {
-    if ( ![self isPluginInitialized] ) return;
-    
-    self.sdk.settings.muted = muted;
+    if ( self.sdk )
+    {
+        self.sdk.settings.muted = muted;
+        self.mutedToSet = nil;
+    }
+    else
+    {
+        self.mutedToSet = @(muted);
+    }
 }
 
 - (BOOL)isMuted
 {
-    if ( ![self isPluginInitialized] ) return false;
+    if ( !self.sdk ) return false;
+    if ( self.mutedToSet ) return self.mutedToSet.boolValue;
```

Comment:
Shouldn't we swap these lines? We should first check `mutedToSet` right, since it would most likely be `nil` is `self.sdk` is not `nil`.